fairscale/utils/parallel.py

```diff
@@ -58,10 +58,14 @@ def enable_pytorch_sync_bn(module: torch.nn.Module) -> None:
             layer._specify_ddp_gpu_num(1)  # type: ignore
 
 
-def get_process_group_cached(ranks: Optional[List[int]] = None) -> ProcessGroup:
+def get_process_group_cached(ranks: Optional[Sequence[int]] = None) -> ProcessGroup:
     """
     Singleton PyTorch distributed group cache. Inspired by the code from fairseq.
 
+    Just like torch.distributed.new_group, this needs to be called on all ranks
+    at the same time when a new group is created, no matter the rank is a member of
```

Comment:
s/",no matter the rank..."/".This is true for all ranks irrespective of their group membership status." or some rephrasing of the last sentence?

Reply:
Thank you! Good point. I will address and merge.